modules/migration-prerequisites.adoc

```diff
@@ -32,6 +33,7 @@ The following `imagestreamtags` have been _removed_ from {product-title} _4.2_:
 * `python:3.3`, `python:3.4`
 * `ruby:2.0`, `ruby:2.2`
 
+ifeval::["{product-version}" == "4.4"]
```

Comment:
Consider using a similar ifeval for the previous section.

Reply:
Not a good idea, because the lists are not cumulative. Someone migrating from 3.7 to 4.4 would still need to know about the tags removed in 4.2.